### PR TITLE
feat(events): add token events

### PR DESCRIPTION
**Pull request by Anouchka-M**
[DOC-89]

- [X] Add a [changelog entry](https://changelog.scalingo.com/)

[DOC-89]: https://scalingo.atlassian.net/browse/DOC-89?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Anouchka-M** [comment]
I don't really get the CI/linter troubles, it seems the errors are kinda the same than on #252 but they didn't cause any trouble on #256

File: events_token.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	TokenId   string `json:"token_id"`
+}
+
+type EventNewTokenType struct {
+	Event
+	TypeData EventNewTokenTypeData `json:"type_data"`
+}
+
+func (ev *EventNewTokenType) String() string {
+	return "New token created"
```
Comment (EtienneM) [inline events_token.go:14]: It would be nice to include the token name in the message

File: events_token.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	TokenId   string `json:"token_id"`
+}
+
+type EventRegenerateTokenType struct {
+	Event
+	TypeData EventRegenerateTokenTypeData `json:"type_data"`
+}
+
+func (ev *EventRegenerateTokenType) String() string {
+	return "Token regenerated"
```
Comment (EtienneM) [inline events_token.go:28]: It would be nice to include the token name in the message

File: events_token.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	TokenId   string `json:"token_id"`
+}
+
+type EventDeleteTokenType struct {
+	Event
+	TypeData EventDeleteTokenTypeData `json:"type_data"`
+}
+
+func (ev *EventDeleteTokenType) String() string {
+	return "Token deleted"
```
Comment (EtienneM) [inline events_token.go:42]: It would be nice to include the token name in the message

**Review by EtienneM — APPROVED** [review]
Thanks \o/

**Pull request merged**